src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -227,20 +227,23 @@ def setup_codemirror_editor(opts = {})
     if @codemirror_editor_setup
       return next_codemirror_uid
     end
-    @codemirror_editor_setup = 0
-    opts.reverse_merge!({ read_only: false, no_border: false, width: 'auto' })
 
+    content_for(:head_style, codemirror_style(opts))
```

Comment:
Both were and are still being used. `content_for_head` is not for the styling part. It's for the inclusion of the JS library. `head_style` is being used for the styling part of the editor (configuration). This didn't change.